RHDEVDOCS-5513: Uninstalling OpenShift Builds (#63619)
openshift/openshift-docs: modules/ob-deleting-the-builds-component-and-custom-resources.adoc

// Module included in the following assemblies:
//
// builds/uninstalling-openshift-builds.adoc

:_content-type: PROCEDURE
[id='ob-deleting-the-builds-component-and-custom-resources_{context}']
= Deleting the {builds-shortname} components and custom resources

Delete the custom resources (CRs) that were created by default during Operator installation.

> 

Reviewer: You mention in the assembly that you have to be a cluster admin to perform the procedures. This should also be included as a Prerequisite for the procedures.

> Delete the Custom Resources (CRs) created by default during installation of the {builds-title} Operator.

Reviewer 2: ```suggestion
Delete the custom resources (CRs) that were created by default when you installed the {builds-title} Operator.
```

.Procedure
. In the *Administrator* perspective of the web console, navigate to *Administration* -> *Custom Resource Definition*.

. Type `operator.shipwright.io` in the *Filter by name* box to search for the {builds-title} Operator CR.

. Click *CRD Config* to see the *Custom Resource Definition Details* page.

> . Click the *Actions* drop-down list and select *Delete Custom Resource Definition*.

Reviewer 2: ```suggestion
. Click the *Actions* drop-down list, and select *Delete Custom Resource Definition*.
```

+
[NOTE]
====

> Deleting the CRs will delete the {builds-title} controller, and all the related components and tasks on the cluster will be removed.

Reviewer 2: ```suggestion
Deleting the CRs deletes the {builds-title} controller, and all the associated components and tasks on the cluster are removed.
```

Reviewer 3: Does this make sense?

```suggestion
Deleting the CRs will delete the {builds-title} controller and all the related components and tasks on the cluster.
```
